src/vis/vis.js

```diff
@@ -319,6 +319,15 @@ var Vis = cdb.core.View.extend({
       this.https = true;
     }
 
+    // this checks that the screen is retina to add @2x to the url
+    if (window.devicePixelRatio > 1) {
+      data.layers.forEach(function(layer){
+        if(layer.type === "tiled"){
```

Comment:
@rochoa Yeah, you're right. This makes me think... is this something we want to add automatically, or let the user activate it with options? Many tile servers won't support an @2x flag.

Leaflet has a ```detectRetina``` option. We can add a similar one to our options and then apply this check.

Regarding the pattern, we can use a regex to add the flag just before the extension dot.